tests/utils/hashing_tests.py

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=no-self-use
+import datetime
+import math
+from typing import Any
+
+import pytest
+
+from superset.utils.hashing import md5_sha_from_dict, md5_sha_from_str
+
+
+def test_basic_md5_sha():
```

Comment:
Maybe add a test to validate consistency between dicts with different sort orders of keys?

Reply:
Cool, added a more explicit test for sort order.